README.md

```diff
@@ -23,15 +27,14 @@ of the SPDX specification, available at: https://spdx.dev/specifications
 tools-golang provides the following packages:
 
 * *spdx* - in-memory data model for the sections of an SPDX document
-* *tvloader* - tag-value document loader
-* *tvsaver* - tag-value document saver
-* *rdfloader* - RDF document loader
-* *json* - JSON document parser and writer
-* *yaml* - YAML document parser and writer
+* *tagvalue* - tag-value document reader and writer
```

Comment:
Is there a preference to use `tagvalue` vs. `tv`?

Reply:
`tagvalue` is fine!